docs/guides/getting-started-agentic-orchestration.md

```diff
@@ -0,0 +1,101 @@
+---
+id: getting-started-agentic-orchestration
+title: "Run your first AI Agent"
+sidebar_label: Run your first AI Agent
+description: "For developers using Camunda 8 Self-Managed, step through an example project with Spring Boot or Node.js."
+keywords: [java, spring, spring camunda, getting started, user guide, tutorial]
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">1 hour</span>
+
+Have you heard about agentic orchestration, but don't know where to start? This guide is tailored for those who want to implement their first agent running on Camunda 8.
+
+## What to expect
+
+You will:
+
+- Use Camunda 8 either
+  - in a local development environment using **Camunda 8 Run** or
+  - use the **SaaS** version where you can model and deploy your processes in the cloud.
+- Use the **Camunda Modeler** to deploy and start a business process.
+- Use the **AI Agent** connector, as the brain for the agent
+- Use an ad-hoc subprocess, as the building block for the agent's behavior
+
+## Prerequisites
+
+- To run an agent with Camunda 8, you need a running environment.
+  - Have a Camunda 8 environment up and running, described in [Run your first local project](../getting-started-example)
+  - Or [sign up for a free SaaS Trial Account](https://accounts.cloud.camunda.io/signup)
+- AWS IAM user with permissions: `bedrock:InvokeModel` (Claude 3 Sonnet/Haiku) and `aoss:*` for your OpenSearch index
+- Email account (SMTP/IMAP) & credentials (App Password for Gmail, or provider-specific credentials)
+- Outbound internet access for connectors to reach email server, Bedrock, and OpenSearch endpoints
+
+## Concept
+
+### Deterministic and dynamic workflows
+
+Traditionally people modeled their business processes as a **deterministic sequence** of steps, where each step is predefined and follows a strict order. This approach works well for simple, repetitive tasks but struggles to adapt to more complex, dynamic scenarios.
```

Comment:
Traditionally business processes are modeled as a **deterministic sequence** of steps, where the flow logic  is predefined and follows a strict order. This approach works well for situations where the flow logic can be defined it advance but struggles to adapt to more complex, open-ended scenarios.